build/jest/envConsole.js

```diff
@@ -0,0 +1,12 @@
+const consoleLogFn = global.console.log;
+
+function consoleLog(message, ...args) {
+    // Workaround tos suppress AJV console log messages from Box3d Runtime
+    if (typeof message === 'string' && message.indexOf('$ref: all keywords') >= 0) {
```

Comment:
There are other `console.log` references in Box3d Runtime, I guess they just don't get executed?

Reply:
Yeah, this is just a temporary workaround. I'm going to try to isolate loading the Box3D SDK to only the 3D/360 tests.